tokio-trace/Cargo.toml

```diff
@@ -15,7 +15,7 @@ keywords = ["logging", "tracing"]
 publish = false
 
 [dependencies]
-tokio-trace-core = "0.1"
+tokio-trace-core = { path = "./tokio-trace-core" }
```

Comment:
Committing this is fine for now, but I think we probably _don't_ want to merge this (or the other changes to update `tokio-trace` to track the breaking change). 

Instead, we probably want to land these changes separately, cut a release of `tokio-trace-core` 0.2, and merge a separate branch updating `tokio-trace` to use `tokio-trace-core` 0.2. 

So, we'll want to back this change out prior to merging this PR.

Reply:
Actually, disregard the above comment --- I spoke to Carl and I think the plan is to use path deps until we release `tokio-trace-core` 0.2 on crates.io. So this is the right thing.